src/App.tsx

```diff
@@ -1,10 +1,56 @@
 import React from 'react';
+// COMPONENTS
+import Content from './components/Content/Content';
+import Block from './components/Block/Block';
+// STYLES
+import styled from 'styled-components';
+
+const AppWrapper = styled.div`
+  background-color: #f6f6f8;
+  display: grid;
+  grid-template-areas: 'b c';
+  width: 1280px;
```

Comment:
these values: 1280px, 1026px and etc should be moved into separate constants (maybe even into separate file)